eclipse/plugins/org.switchyard.tools.ui.editor/src/org/switchyard/tools/ui/editor/components/http/HttpProxyComposite.java

```diff
@@ -126,47 +126,47 @@ protected void handleUndo(Control control) {
 
     private void bindControls(final DataBindingContext context) {
         final EditingDomain domain = AdapterFactoryEditingDomain.getEditingDomainFor(getTargetObject());
-        final Realm realm = SWTObservables.getRealm(_proxyHostText.getDisplay());
+        final Realm realm = DisplayRealm.getRealm(_proxyHostText.getDisplay());
 
-        _bindingValue = new WritableValue(realm, null, HTTPBindingType.class);
-        final IObservableValue hostValue = new WritableValue(realm, null, String.class);
-        final IObservableValue passwordValue = new WritableValue(realm, null, String.class);
-        final IObservableValue portValue = new WritableValue(realm, null, String.class);
-        final IObservableValue userValue = new WritableValue(realm, null, String.class);
+        _bindingValue = new WritableValue<>(realm, null, HTTPBindingType.class);
+        final IObservableValue<String> hostValue = new WritableValue<String>(realm, null, String.class);
```

Comment:
I mentioned it only on previous line before but you can write new WritableValue<String> also for all these lines.